### ISSUE TITLE
Ability to force SSL on certain pages

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vosaocms/vosao#293]
```
There needs to be the ability to force certain pages to use SSL, and other's to 
be forced to use standard http. The scenario comes around when:

1. You have a site like http://www.mysite.com
2. Some of those pages require SSL (for passwords or something, whatever). 
Force SSL (that means force certain pages to go to https://appid.appspot.com/). 
This can currently done by modifying web.xml but is a clunky way of doing 
things.
3. You don't want Googlebot to crawl both your http://www.mysite.com and your 
http://appid.appspot.com thinking they are different sites. So force standard 
http on non-ssl pages.

Basically, the only reason I (and I guess others) is so Googlebot doesn't crawl 
and index the same site twice.
```

Original issue reported on code.google.com by `roland.q...@gmail.com` on 31 Jul 2010 at 6:05
